sao/src/view.js

```diff
@@ -157,6 +157,14 @@
                     }
                 }
             }
+
+            if (['field', 'label'].includes(node.tagName) &&
+                jQuery.isEmptyObject(field)) {
+                Sao.Logger.error(
```

Comment:
There is no actual error here (meaning, no pop-up, just the log).

I am not sure of what I'd rather have, but the Gtk client raises an error.